wiki/content/deploy/tls-configuration.md

```diff
@@ -121,13 +150,16 @@ Important points:
 * Node certificates are only valid for the hosts listed.
 * Client certificates are only valid for the named client/user.
 
-## TLS Options
+## TLS options
 
-The following configuration options are available for Alpha:
+The following configuration options are available for Dgraph Alpha and Dgraph
+Zero nodes:
 
 * `--tls_dir string` - TLS dir path; this enables TLS connections (usually 'tls').
 * `--tls_use_system_ca` - Include System CA with Dgraph Root CA.
-* `--tls_client_auth string` - TLS client authentication used to validate client connection. See [Client Authentication Options](#client-authentication-options) for details.
+* `--tls_client_auth string` - TLS client authentication used to validate client
+  connections from external ports. To learn more, see
```

Comment:
tls_internal_port_enabled flag is available in 20.07. This specify whether to connect with internal ports with tls config or not